client/my-sites/plan-features/index.jsx

```diff
@@ -537,6 +542,16 @@ export class PlanFeatures extends Component {
 						primaryUpgrade={ primaryUpgrade }
 						selectedPlan={ selectedPlan }
 					/>
+					{ isUnlaunchedAndGoingAtomic && (
```

Comment:
This needs to apply to private sites across the board -- not just those which are "unlaunched"

Reply:
This is done.